tests/test_logger.py

```diff
@@ -553,14 +553,14 @@ def test_rollout_success_rate_on_policy_algorithm(tmp_path):
     env = Monitor(DummySuccessEnv(dummy_successes, ep_steps), filename=monitor_file, info_keywords=("is_success",))
 
     # Equip the model of a custom logger to check the success_rate info
-    model = PPO("MlpPolicy", env=env, stats_window_size=STATS_WINDOW_SIZE, n_steps=env.steps_per_log, verbose=1)
+    model = PPO("MlpPolicy", env=env, stats_window_size=STATS_WINDOW_SIZE, n_steps=env.env.steps_per_log, verbose=1)
```

Comment:
better to define this variable before then `steps_per_log = env.unwrapped.steps_per_log`.
Using `env.unwrapped` is probably cleaner.

Reply:
Good point, will do